### DISCUSSION TITLE
About server resource usage in akhq

### DISCUSSION THREAD

**Discussion by Qyuee** [tchiotludo/akhq#679] [Q&A] [answered]
I think using akhq will slightly increase the disk I/O level of the server.
Is there any way to prevent or limit this?
Or do we not have to consider that?

**Comment by tchiotludo** [comment, marked as answer]
Clearly, AKHQ fetch server metadata and will have an impact on the broker. 
Mostly, it must not be huge since we are only talking about metadata. 
* Do you have some scenario where you can see the increase of resource ? 
* Maybe some page on AKHQ (topics list, consumer groups, ...) where you can capture unusual load on the broker ?

**Reply by Qyuee** [reply]
Thank you for your response.
If you search for a topic message through a certain value through aakhq, it seems to occupy quite a cpu. If multiple administrators were searching for topic messages, I wondered if overusing the broker server's cpu resources might cause omnipotent server instability. The cpu usage of the server was checked through the ‘top’ command.

**Reply by tchiotludo** [reply]
Seems to be weird, It's kafka main feature to be able to consume the same topic by many users. 
Clearly it may consume resource (reading from disk and network usage mostly), but just few admin can't crash a Kafka server. 
Always depends on your workload and sizing of your cluster
